app/javascript/components/EditPopup/EditPopup.js

```diff
@@ -48,6 +48,7 @@ const EditPopup = (props) => {
     });
   };
   const isLoading = isNil(task);
+  const handleFormChange = setTask;
```

Comment:
should be something like `const handleFormChange = task => setTask(task);`

Reply:
Can you explain the difference between const `handleFormChange = setTask` and `const handleFormChange = (task) => setTask(task)`? As I understand it, the props gets a reference to the function in either case.